src/window/config.c

```diff
@@ -608,9 +608,9 @@ static const uint8_t *display_text_resolution(void)
 {
     uint8_t *str = display_text;
     resolution *r = &available_resolutions[data.config_values[CONFIG_ORIGINAL_WINDOWED_RESOLUTION].new_value];
-    str += string_from_int(str, r->width, 0);
+    str += string_from_int(str, r->width, 0, 0);
```

Comment:
Hummm. Should we use the separator for the resolutions? Not really sure though...

Reply:
You don't typically see resolutions written with comma. `"1024x768"` etc. We can certainly add it if you want, but I think it's non-standard.